packages/nameguard-react/src/components/Report/ReportFormattedName.tsx

```diff
@@ -1,26 +1,28 @@
 import React from "react";
+import { DisplayedName } from "../DisplayedName/DisplayedName";
+import { ENSName } from "@namehash/ens-utils";
+import { Normalization } from "@namehash/nameguard";
 
 type Props = {
+  name: ENSName;
```

Comment:
The only field we should pass here is an `ENSName`. All the other fields in these Props should be removed. The other props already exist inside the provided `ENSName`.